slack-market/api/endpoints.rb

```diff
@@ -1,4 +1,5 @@
 require 'slack-market/api/endpoints/teams_endpoint'
 require 'slack-market/api/endpoints/subscriptions_endpoint'
 require 'slack-market/api/endpoints/status_endpoint'
+require 'slack-market/api/endpoints/slack_endpoint'
```

Comment:
Should be called `graph_endpoint.rb`. 

I think you had a good point wrt making it "slack" vs. "graph", but there're two things that are difficult in computer science, caching and naming :) I prefer domain-specific names, in my experience they just always last longer.